daemon/daemon.go

```diff
@@ -1278,6 +1268,17 @@ func NewDaemon() (*Daemon, *endpointRestoreState, error) {
 	if err := node.ValidatePostInit(); err != nil {
 		log.WithError(err).Fatal("postinit failed")
 	}
+
+	if k8s.IsEnabled() {
+		log.Info("Annotating k8s node with CIDR ranges")
+		err := k8s.AnnotateNode(k8s.Client(), node.GetName(),
```

Comment:
Don't we need to return a channel and block before ipam init?

Reply:
@tgraf ipam.Init() is executed on line 1234